dev/how-to/maintain/upgrade/to-tidb-3.0.md

```diff
@@ -0,0 +1,140 @@
+---
+title: TiDB 3.0 Upgrade Guide
+summary: Learn how to upgrade to TiDB 3.0
+category: upgrade
+---
+
+# TiDB 3.0 Upgrade Guide
+
+This document is targeted for users who want to upgrade from TiDB 2.0 (above V2.0.1) or TiDB 2.1 RC to TiDB 3.0. TiDB 3.0 is compatible with [TiDB Binlog of Kafka Version](/dev/tools/binlog/tidb-binlog-kafka.md) and [TiDB Binlog of Cluster Version](/dev/tools/binlog/overview.md).
```

Comment:
Since Morgan renamed the corresponding files in #1123, the links need to be updated again:

```suggestion
This document is targeted for users who want to upgrade from TiDB 2.0 (above V2.0.1) or TiDB 2.1 RC to TiDB 3.0. TiDB 3.0 is compatible with [TiDB Binlog of Kafka Version](/dev/reference/tools/tidb-binlog/tidb-binlog-kafka.md) and [TiDB Binlog of Cluster Version](/dev/reference/tools/tidb-binlog/overview.md).
```